stonesoup/predictor/tests/test_multimodel.py

```diff
@@ -0,0 +1,140 @@
+import datetime
+
+import numpy as np
+
+from ...models.transition.linear import (
+    ConstantVelocity, CombinedLinearGaussianTransitionModel, ConstantAcceleration)
+from ...predictor.particle import MultiModelPredictor
+from ...types.particle import MultiModelParticle
+from ...types.state import MultiModelParticleState
+
+
+def test_multi_model():
+
+    # Define time related variables.
+    timestamp = datetime.datetime.now()
+    time_diff = 2  # 2sec.
+    new_timestamp = timestamp + datetime.timedelta(seconds=time_diff)
+
+    # Define prior state.
+    prior_particles = [
+        MultiModelParticle(np.array([[10], [10], [10], [10], [10], [10], [10], [10], [10]]),
```

Comment:
Seeing this huge list is making me think maybe it would be worth having a container class for Particle/MultiModelParticle, then perhaps a function that could generate something like this given a few parameters?  I'm not sure how that would work specifically, but this kind of thing definitely contravenes the standard coding advice about avoiding excessive repetition right?  Maybe a change that would be worth a new separate PR.  Cue "Nicely volunteered"!

Reply:
`MultiModelParticleState` has also been added in this PR, which like `ParticleState` lets you just use `(no. dimension, no. particles)` arrays. I could update this test to use it.

Reply:
Oh cool, that was what I was looking for.  Yeah I reckon that'd be worth doing, as I know I often turn to tests as a way of seeing how the components are intended to be used.  If that is the case